sound/soc/codecs/rt711.c

```diff
@@ -544,9 +544,12 @@ static int rt711_set_amp_gain_put(struct snd_kcontrol *kcontrol,
 			break;
 	}
 
+	mutex_lock(&rt711->calibrate_mutex);
```

Comment:
@shumingfan Does rt711_set_bias_level() needs the mutex, too?

Reply:
@bardliao Right, rt711_set_bias_level() should need the mutex. Thanks.

Reply:
@bardliao if you want to close that PR and add a new one with more fixes that's fine with me. I am running out of time.